src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
```

Comment:
spacing around '+'

Reply:
> hmm..
> Slider does not work in IE, at all.. and it wont, if we want to use native input element.
> I was checking also the library imodel.js uses and it is done without native slider. This gives ability to fully control stuff and add what ever we want versus fighting with the limitation of native input (now range slider needs to use actually 2 elements...). Even w3 example is not using native input element https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-1.html We could take idea from it, apply all the accessibility stuff, etc..
> So I am thinking, should we just do the styling here with simple divs and implement functionality itself in js?
> Especially, if there is going to be some more advanced stuff required, we would not need to refactor everything again...
> 
> ![image](https://user-images.githubusercontent.com/81580355/124270913-78bfe380-db45-11eb-8f13-171c26d40ae6.png)

@FlyersPh9 @veekeys - That is definitely an option.  I was basing my implementation on what was done in the original Azure repo.  We need to make a decision on which way to proceed as soon as possible. We keyboard and touch processing for free with the current implementation. If we go to totally js solution does that need to be implemented in both backstop and iTwinUI-react?